packages/client/src/components/PlayButton/PlayButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import style from './button.module.css'
+
+const PlayButton = () => {
+  return (
+    <button className={style.button}>
+      <div className={style.circle} />
+      <svg
```

Comment:
Иконка есть в `@assets/icon`. Здесь как компонент лучше использовать (если это она конечно, если нет то перенести туда)

```suggestion
      <PlayIcon />
```